code/controllers/configuration/entries/general.dm

```diff
@@ -70,6 +70,8 @@
 /datum/config_entry/flag/log_law/DeprecationUpdate(value)
 	return value
 
+/datum/config_entry/flag/log_tools // log using tools on machines
```

Comment:
```suggestion
/// Log usage of tools
/datum/config_entry/flag/log_tools
```

1. Real autodoc
2. It has no reason to stay machine specific (if it even is now)